CasterTrainer.py

```
@@ -0,0 +1,322 @@
+                API.Pause(spell["wait"] / 1000.0)
+                _update_gump(school)
+                break
+
+
+# Start button callback: read caps and begin training.
+def _on_start():
+    global RUNNING
+    for name, tb in TEXT_INPUTS.items():
+        try:
+            SPELL_CAPS[name] = int(tb.Text.strip()) if tb.Text else 0
+        except Exception:
```

Ruff (0.14.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)